src/entities/fractions/fraction.ts

```diff
@@ -52,6 +52,16 @@ export class Fraction {
     return new Fraction(this.denominator, this.numerator)
   }
 
+  public absoluteValue(): Fraction {
+    const numeratorAbs = JSBI.lessThan(this.numerator, JSBI.BigInt(0))
```

Comment:
`I discovered a bug with how I was doing absolute value in there. Fraction.lessThan() is broken when comparing negative fraction to 0, as multiplying by 0 will cancel out the negative number.`

Reply:
I can take a stab at fixing `lessThan`, but I went to create `absoluteValue` even before I knew there was a `lessThan` bug. Reason for keeping this lib super minimal?

Reply:
cool, guess I misunderstood then. tbh was just feeling it out if for you to suggest that I fix lessThan, in case you had some weird reason for keeping it and not supporting negative numbers or something.